### PR TITLE
Add policy regarding live test

### PR DESCRIPTION
**Pull request by troydai**
Also clean up the markdown lint errors.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/6308
This is an **experimental preview** for @microsoft users.

**Comment by troydai** [comment]
Will not wait for the CI since it is documentation change.

File: doc/authoring_tests.md [hunk 1]
```diff
@@ -1,27 +1,44 @@
-Automating tests for Azure CLI
-========================================
+# Automating tests for Azure CLI
 
 ## Overview
 
 There are two types of automated tests you can add for Azure CLI: [unit tests](https://en.wikipedia.org/wiki/Unit_testing) and  [integration tests](https://en.wikipedia.org/wiki/Integration_testing).
 
-For unit tests, we support unit tests written in the forms of both standard [unittest](https://docs.python.org/3/library/unittest.html) and [nosetest](http://nose.readthedocs.io/en/latest/writing_tests.html).
+For unit tests, we support unit tests written in the forms standard [unittest](https://docs.python.org/3/library/unittest.html). The current support of [nosetest](http://nose.readthedocs.io/en/latest/writing_tests.html) will be gradually retired.
```
Comment (tjprescott) [inline doc/authoring_tests.md]: This grammar is wonky.  Do we even have any tests written in "nosetest form"?  If not, we can just erase mention of it.

File: doc/authoring_tests.md [hunk 2]
```diff
@@ -1,27 +1,44 @@
...
 
-* Do not use hard-coded or otherwise persistent resources. This makes it difficult or impossible to re-record tests or run them live. In general, all resources needed to support the test should be created as part of the test and torn down after the test finishes. The use of `ResourceGroupPreparer` helps to facilitate this.
-* Tests _must_ be included for all new command modules and any new commands to existing test modules. PRs will be rejected outright if they do not include tests.
+* __DO NOT USE__ hard-coded or otherwise persistent resources. This makes it difficult or impossible to re-record tests or run them live. In general, all resources needed to support the test should be created as part of the test and torn down after the test finishes. The use of `ResourceGroupPreparer` helps to facilitate this.
+* Tests __MUST__ be included for all new command modules and any new commands to existing test modules. PRs will be rejected outright if they do not include tests.
 * Name test methods in the following format: `test_<module>_<feature>`.
+* The scenario test must be able to run repeatly in live mode. The feature own is responsible of maintaining their scenario tests.
```
Comment (tjprescott) [inline doc/authoring_tests.md]: Typo:  "repeatly" > "repeatedly".   "feature own" > "feature owner".

File: doc/authoring_tests.md (hunk 1 above)
Comment (troydai) [reply doc/authoring_tests.md]: I will remove it then.

File: doc/authoring_tests.md (hunk 2 above)
Comment (troydai) [reply doc/authoring_tests.md]: Will fix.

**Comment by troydai** [comment]
Updated.

**Pull request merged**